From 6f1f02dacef70b464746742d8af5fb39cc410531 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Fri, 26 Dec 2014 12:50:55 +0800
Subject: [PATCH 419/602] MA-5998 change the permission of file system with
 tool mkfs_ubifs

the option "-F" add the capability of setting file system for tool mkfs_ubifs

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 imx6/imx6.mk                 | 2 +-
 sabreauto_6q/BoardConfig.mk  | 2 +-
 sabreauto_6sx/BoardConfig.mk | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index ae705f7..8d9c9a4 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -289,7 +289,7 @@ omx_excluded_libs :=					\
 
 PRODUCT_PACKAGES += $(omx_libs) $(omx_excluded_libs)
 
-PRODUCT_PACKAGES += libubi ubinize ubiformat ubiattach ubidetach ubiupdatevol ubimkvol ubinfo mkfs.ubifs
+PRODUCT_PACKAGES += libubi ubinize ubiformat ubiattach ubidetach ubiupdatevol ubimkvol ubinfo mkfs_ubifs 
 
 # FUSE based emulated sdcard daemon
 PRODUCT_PACKAGES += sdcard
diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index 1b41b17..0d22551 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -87,7 +87,7 @@ BOARD_KERNEL_CMDLINE := console=ttymxc3,115200 init=/init video=mxcfb0:dev=ldb,b
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
 UBI_ROOT_INI := device/fsl/sabreauto_6q/ubi/ubinize.ini
-TARGET_MKUBIFS_ARGS := -m 4096 -e 253952 -c 4096 -x none
+TARGET_MKUBIFS_ARGS := -m 4096 -e 253952 -c 4096 -x none -F
 TARGET_UBIRAW_ARGS := -m 4096 -p 256KiB $(UBI_ROOT_INI)
 
 # Note: this NAND partition table must align with MFGTool's config.
diff --git a/sabreauto_6sx/BoardConfig.mk b/sabreauto_6sx/BoardConfig.mk
index c8be829..0b91e35 100755
--- a/sabreauto_6sx/BoardConfig.mk
+++ b/sabreauto_6sx/BoardConfig.mk
@@ -71,7 +71,7 @@ BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=tt
 ifeq ($(TARGET_USERIMAGES_USE_UBIFS),true)
 #UBI boot command line.
 UBI_ROOT_INI := device/fsl/sabreauto_6sx/ubi/ubinize.ini
-TARGET_MKUBIFS_ARGS := -m 4096 -e 253952 -c 4096 -x none
+TARGET_MKUBIFS_ARGS := -m 4096 -e 253952 -c 4096 -x none -F
 TARGET_UBIRAW_ARGS := -m 4096 -p 256KiB $(UBI_ROOT_INI)
 
 # Note: this NAND partition table must align with MFGTool's config.
-- 
1.8.0

